README.md

```diff
@@ -1,50 +1,150 @@
 1. Create a new branch in your repo
 2. cd into folder above
 3. run: `cookiecutter -f -s https://github.com/DSACMS/repo-scaffolder --directory=tierX`
 4. Make sure when answering the questions you use the existing folder/project name
 5. Raise pr into main
 
-## Updating Projects
-When creating projects, if you want to receive updates then add `dsacms-tierX` as a github topic to the repo.  The scaffolder repo includes github workflows that will find all repos with that tag and can raise a pull request with an updated string or adding a file.  See [actions.md](https://github.com/DSACMS/repo-scaffolder/blob/main/.github/actions.md) for more information.
+### Metadata collection using code.json
 
-## Editing or Adding Tiers
-At a top level, each tier consists of a folder for `hooks`, a folder containing the files to be added (`{{cookiecutter.project_slug}}`), and a `cookiecutter.json` defining the questions cookiecutter asks.  These naming conventions must be 
-followed as that is what cookiecutter picks up.  The `hooks` folder needs to be duplicated in each tier.  The folder 
-containing the files to be added can include slugged out variables such as `{{ cookiecutter.project_name }}` that can 
-be filled in by the answers to `cookiecutter.json`.  For example, `{{ cookiecutter.project_name }}` will be filled in by 
-this question - `"project_name": "My Project",`.  See the [cookiecutter docs](https://cookiecutter.readthedocs.io/en/stable/) 
-for more information.
+<!-- TODO: Add text here about code.json and link code.json docs -->
+
+#### Add code.json to your project
 
-## Repolinter
+Each repository should contain a code.json file with metadata about the project.
+
+To add code.json into your project, navigate to your project's `.github` directory and run the following cookiecutter command. You will be asked questions about the project (see cookiecutter.json) in order to collect and store this metadata in code.json.
+
+```
+cookiecutter . --directory=codejson
+```
+
+### Maintaining your repository using repo-scaffolder
+
+#### Updating projects with new repo-scaffolder upstream file changes
+
+When creating projects, if you want to receive updates then add `dsacms-tierX` as a github topic to the repo. The scaffolder repo includes github workflows that will find all repos with that tag and can raise a pull request with an updated string or adding a file. See [actions.md](https://github.com/DSACMS/repo-scaffolder/blob/main/.github/actions.md) for more information.
+
+### Identify missing files and information using repolinter
+
+<!-- TODO: Revise once repolinter GH actions are up and link to it-->
 
 Repolinter is a tool maintained by the [TODOGroup](https://todogroup.org/) for checking repositories for common open source issues, using pre-defined rulesets. This can be run stand-alone as a script, pre-commit in your IDE, or post-commit or within CI/CD systems!
```

Comment:
@sachin-panayil For the repolinter section, feel free to revise and add info about the gh actions. This can be done in this branch or in a later PR, no rush